apps/condo/domains/acquiring/utils/validate/bic.utils.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Validator BIC
+ *
+ * The following checks are performed:
+ * 1) Сhecking for emptiness
+ * 2) Checking for length and format (3 digits, then 2 digits or capital letters, and 3 more digits)
+ * 3) Сhecking the country code for compliance
+ *
+ * Example:
+ * BIC - 606000987
+ */
+
+const EMPTY = 'Bic is empty'
+const NOT_NUMERIC = 'Bic can contain only numeric digits'
+const WRONG_LENGTH = 'Bic length was expected to be 9, but received '
+const WRONG_COUNTRY = 'For RU organizations country code is 04, but bic have '
+
+
+class RecipientBicValidation {
+
+    errors = []
+
+    validateForNumbersAndLength (bic) {
+        if (/[^0-9]/.test(bic)) {
+            this.errors.push(NOT_NUMERIC)
+        }
+        if (bic.length !== 9) {
+            this.errors.push(WRONG_LENGTH + bic.length)
+        }
+    }
+
+
+    validateBic (bic, country) {
+        const bicWithoutSpaces = bic.toString().trim()
+
+        if (!bic.length) {
+            this.errors.push(EMPTY)
+        }
+
+        if (country === 'ru') {
+            const countryCode = bicWithoutSpaces.substr(0, 2)
+            if (countryCode !== '04') {
+                this.errors.push(WRONG_COUNTRY + countryCode)
+            }
+
+            this.validateForNumbersAndLength(bicWithoutSpaces)
+        }
+
+        return {
+            result: !this.errors.length,
+            errors: this.errors,
+        }
+    }
+}
+
+const validateBic = (bic, country = 'ru') => {
```

Comment:
You can't validate without a country. It's required! You don't need to hardcode `ru`!